File: test/testTransSegue/testTransSegue/ViewController.m
//
//  ViewController.m
//  testTransSegue
//
//  Created by caoyuan on 7/13/14.
//  Copyright (c) 2014 caoyuan. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
            
- (IBAction)btaction:(id)sender;

@end

@implementation ViewController
            
- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)push:(id)sender {
    [self performSegueWithIdentifier:@"good" sender:sender];
}
- (IBAction)btaction:(id)sender {
}
- (IBAction)dsfs:(id)sender {
}
@end

File: test/dataTrans/dataTrans/ViewController.swift
//
//  ViewController.swift
//  dataTrans
//
//  Created by caoyuan on 7/12/14.
//  Copyright (c) 2014 caoyuan. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var tempString:String = ""
    @IBOutlet var textFiedl : UITextField
    @IBOutlet var tfeld : UITextField
    @IBOutlet var textField : UITextField
    @IBAction func push(sender : AnyObject) {
        var sVC = SViewController()
        sVC.tempString = self.textField.text
//[self performSegueWithIdentifier:@"111" sender:self];
        self.performSegueWithIdentifier("good", sender: sender)
//        self.shouldPerformSegueWithIdentifier("good", sender:self);
    }
    @IBOutlet var tf : UITextField
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

